<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiWindow</class>
 <widget class="QMainWindow" name="GuiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Robot</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>271</width>
           <height>248</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_Comport">
              <property name="text">
               <string>COM PORT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_Comport"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_Baudrate">
              <property name="text">
               <string>BAUD RATE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_Baudrate"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Connect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_Velocity">
                <property name="text">
                 <string>VELOCITY</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sliderVelocity">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_Accelerate">
                <property name="text">
                 <string>ACCERLERATE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sliderAccerlerate">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Home">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Scan">
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Stop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_TestMoveJoint">
            <property name="text">
             <string>Test Move Joint</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>411</width>
           <height>311</height>
          </rect>
         </property>
         <property name="title">
          <string>Camera</string>
         </property>
         <widget class="QPushButton" name="pushButton_StartCamera">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>190</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#pushButton_StartCamera
{
 border:none;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_SetROI">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Select  ROI</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_Position">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>110</y>
            <width>104</width>
            <height>71</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>262</width>
            <height>271</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_Camera">
             <property name="minimumSize">
              <size>
               <width>260</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>260</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>5</number>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_Pick">
               <property name="text">
                <string>PICK</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Hold">
               <property name="text">
                <string>HOLD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Place">
               <property name="text">
                <string>PLACE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <zorder>pushButton_Place</zorder>
         <zorder>pushButton_Pick</zorder>
         <zorder>pushButton_SetROI</zorder>
         <zorder>pushButton_StartCamera</zorder>
         <zorder>label_Camera</zorder>
         <zorder>label_Camera</zorder>
         <zorder>textEdit_Position</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
